src/BoilerPlate/App.cpp

```diff
@@ -20,6 +14,15 @@ namespace Engine
 		, m_timer(new TimeManager)
 		, m_mainWindow(nullptr)
 	{
+		playerONE  = Player(m_width, m_height);
+
+		asteroidSmall = Asteroids(Asteroids::SMALL_SIZE, m_width, m_height);
```

Comment:
This should be abstracted out to a function. Still same thing about naming convention as described above.